lib/postgres.js

```diff
@@ -47,14 +47,100 @@ class Postgres {
             stickers    jsonb,
             updated     timestamp NOT NULL,
             rarity      smallint NOT NULL,
+            floatid     bigint  NOT NULL,
             PRIMARY KEY (a)
-        )`);
+        );
+
+        CREATE TABLE IF NOT EXISTS history (
+            floatid     bigint  NOT NULL,
```

Comment:
This, and other mutli-word columns, should be underscored as per the naming convention but a bit late for that now.

Reply:
Other fields aren't, including the proto definitions they are based on, so I thought it'd be better to follow that standard.

Reply:
Yeah I understand.